### ISSUE TITLE
Repeatable results

### ISSUE THREAD

**Issue by briantcantwell** [nerfstudio-project/nerfstudio#902]
Hello!

I'm noticing that if I run nerfacto on the same dataset over and over again, I get pretty significant differences with each run.   Should that be expected?

**Comment by tancik** [comment]
Some variance is expected to some extent due to the random nature of the optimization. "Sigificant" differences is a bit surprising and should be looked into a bit more. I wonder if a random seed isn't being set somewhere...

**Comment by briantcantwell** [comment]
Right, I didn't expect it to match exactly, but the differences are more extreme than I would have thought.  Thanks for looking into it!

**Comment by thuanaislab** [comment]
any update on this? I got the same issue, even trying to set all the seed at the beginning
